tests/Models/Article.php

```diff
@@ -41,7 +41,7 @@ public function category()
 
     public function tags()
     {
-        return $this->belongsToMany(Tag::class);
+        return $this->morphToMany(Tag::class, 'taggable');
```

Comment:
Would this change still cover all the use cases for a simple `belongsToMany` relationship?